tabs.js

```diff
@@ -0,0 +1,92 @@
+$(function () {
```

Comment:
Please translate this code to vanilla JS. I haven't used jQuery anywhere in the repo because it slows down the process and I'd want consistency in the code. This shouldn't take too long :)